src/audio/google_rtc_audio_processing.c

```diff
@@ -275,6 +410,8 @@ static struct comp_dev *google_rtc_audio_processing_create(
 	if (cd) {
 		rfree(cd->output_buffer);
 		rfree(cd->aec_reference_buffer);
+		GoogleRtcAudioProcessingDetachMemoryBuffer();
```

Comment:
where is `cd->state` cleaned up?

Reply:
Good find! It is not at all cleaned up here, and has never been.

I added a conditional cleanup.
PTAL.